ls1intum/Artemis
# `Quiz exercises`: Fix file input button color for dark mode when creating drag and drop question

### Checklist
#### General


#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


The drag and drop file button("Choose file") is unclear under dark mode.
### Description

Set the values of two variables of bootstrap form-control to make it clear.
### Steps for Testing

Prerequisites:
- 1 Instructor

1. Go to create a quiz exercise
2. Click "Add drag and drop question"
3. Check if the "Choose File" button looks good under dark mode

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Screenshots


Before:
![image](https://github.com/ls1intum/Artemis/assets/60965100/9ce5cab1-2d0c-4e55-98a9-e8a9edd9dfde)
After:
![image](https://github.com/ls1intum/Artemis/assets/60965100/0f31742c-0d01-4b03-ad78-7253147b84b1)
